docs/digger/admin.asciidoc

```diff
@@ -5,12 +5,12 @@ title: Digger Administration guide
 toc_generate: true
 ---
 
-include::docs/digger/admin/monitoring.asciidoc[]
+include::/admin/digger-nagios-monitoring.asciidoc[]
 
-include::docs/digger/admin/use-http-proxy.asciidoc[]
+include::/admin/using-http-proxy.asciidoc[]
 
-include::docs/digger/admin/manage-artifacts.asciidoc[]
+include::/admin/managing-digger-logs-artifacts.asciidoc[]
 
-include::docs/digger/admin/installing-plugins.asciidoc[]
+include::/admin/installing-jenkins-plugins.asciidoc[]
 
-include::docs/digger/admin/known-issues.asciidoc[]
+include::/admin/known-issues.asciidoc[]
```

Comment:
@pmacko1 you should follow the instruction here to preview them: https://github.com/aerogear/aerogear.org#building.

I believe the `docs/` prefix is required.